pkgs/development/tools/phpactor/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenvNoCC, fetchFromGitHub, php, phpPackages }:
+{ lib, stdenvNoCC, fetchFromGitHub, makeWrapper, php, phpPackages }:
```

Comment:
Could you use makeBinaryWrapper instead ?

Reply:
For sure, just curious what the reason is to prefer makeBinaryWrapper over makeWrapper? I couldn't find much in the way of documentation

Reply:
It has much less issues when being used on Darwin.